api/index.md

```diff
@@ -4,39 +4,31 @@ title: API
 nav_order: 20
 has_children: true
 ---
+
 # Overview
 
 The [Data Commons Graph](https://datacommons.org) aggregates data from many
 different [data sources](https://datacommons.org/datasets) into a single
 database. Data Commons is based on the data model used by
 [schema.org](https://schema.org); for more information, see [the guide to the data model](/data_model.html).
 
-The **Data Commons API** allows developers to programmatically access the data in Data Commons.
+The Data Commons APIs allow developers to programmatically access the data in Data Commons.
 Data Commons provides several different ways to access its API's resources:
 
-1. A [REST API](/api/rest/v2) that can be used on the command line as well as in any language with an HTTP library.
-1. A lightweight [Python](/api/python) wrapper.
-1. A heavier [Pandas](/api/pandas) wrapper.
-1. A convenient [Google Sheets](/api/sheets) add-on.
+* A [REST API](/api/rest/v2) that can be used on the command line as well as in any language with an HTTP library.
+* A lightweight [Python](/api/python) wrapper.
+* A heavier [Pandas](/api/pandas) wrapper.
 
-The endpoints can be roughly grouped into four categories.
+The endpoints can be roughly grouped into four categories:
```

Comment:
I wonder if should reorder them. maybe
1. stats data
2. graph exploration
3. graph query
4. utilities

Reply:
Yes and in that case, maybe we should reorder the endpoints in the navigation too.